requirements.txt

```diff
@@ -0,0 +1,23 @@
+alabaster==0.7.12
```

Comment:
In most openedx projects we use a pattern of having `requirements.in` files that generate the final `requirements.txt` using pip-tools.  I think this is a good pattern and we should continue to use that here.  You can take a look at this is done in for python projects in the [edx-cookiecutter](https://github.com/edx/edx-cookiecutters/tree/master/python-template/%7B%7Bcookiecutter.placeholder_repo_name%7D%7D) repo.  Let me know if you have more questions about this.

Reply:
I've added requirements.in, requirements.txt, and a new makefile that can now `make requirements` and `make upgrade`, similarly to the makefile in edx-documentation.